libraries/AP_Param/AP_Param.cpp

```diff
@@ -2519,16 +2519,16 @@ void AP_Param::show(const AP_Param *ap, const char *s,
 {
     switch (type) {
     case AP_PARAM_INT8:
-        port->printf("%s: %d\n", s, (int)((AP_Int8 *)ap)->get());
```

Comment:
Should this be pulled out into a separate PR?

Reply:
It's fairly fundamental to the mapping I'm using